<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRun</class>
 <widget class="QWidget" name="FormRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Program Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPrgName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtStatus"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnCycle">
         <property name="text">
          <string>Cycle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtCycle">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tbWidget">
       <property name="currentIndex">
        <number>8</number>
       </property>
       <widget class="QWidget" name="tbmain">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub1">
        <attribute name="title">
         <string>Sub1</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub2">
        <attribute name="title">
         <string>Sub2</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub3">
        <attribute name="title">
         <string>Sub3</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub4">
        <attribute name="title">
         <string>Sub4</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub5">
        <attribute name="title">
         <string>sub5</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub6">
        <attribute name="title">
         <string>sub6</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub7">
        <attribute name="title">
         <string>sub7</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tbsub8">
        <attribute name="title">
         <string>sub8</string>
        </attribute>
        <widget class="QTableView" name="tblRunSteps_Sub8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>391</height>
          </rect>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConProgram">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetStep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStepRun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Step Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBstepRun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Step BRun</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAbortRun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Abort Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSpeed">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
